source/adapters/level_zero/event.cpp

```diff
@@ -1033,6 +1024,16 @@ ur_result_t urEventReleaseInternal(ur_event_handle_t Event) {
     }
   }
 
+  // Save pointer to the queue before deleting/resetting event.
```

Comment:
I think this comment was for this line:
`auto Queue = Event->UrQueue;`
:)

Reply:
thanks, I will move that comment.

Reply:
fixed